helm/prometheus-rules/templates/recording-rules/resource-usage.rules.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: resource-usage.recording.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: monitoring.resource-usage.recording
```

Comment:
I'm not happy about the naming. I think we should have them under something like giantswarm:observability:monitoring:resource_usage. WDYT?

Reply:
What about `giantswarm:observability:monitoring:memory_estimation` ?
- `resources` is not specific enough
- it's not real usage, it's an estimation and we don't know yet how well it maps to real usage

Reply:
Well, resources in kubernetes land is quite self-explanatory though. but estimation or estimate makes a lot of sense yes

Reply:
what about giantswarm:observability:monitoring:resource_usage_estimations instead?

Reply:
oh, I just saw that there was one more sub-key named `memory_working_set_bytes`, so `resources` makes sense yes :facepalm: Sorry.

Reply:
No worries :) Who knows, maybe later we will use it for other things 🤔